src/whylogs/app/session.py

```diff
@@ -327,10 +328,12 @@ def close(self):
 
         self._active = False
         loggers = list(self._loggers.items())
-        for name, logger in loggers:
-            if logger.is_active():
-                logger.close()
-            self.remove_logger(name)
+        with tqdm(loggers) as t:
```

Comment:
This will polutes user's stdout!! Imagine running a web application with expected logging format in stdout, only to be polluted by tqdm in non-interactive mode!

A better solution is to detect if you are in a Jupyter notebook and attempt to import tqdm at that point, and if not available, should just not fail the program but could just print out some warning about no progress bar due to `tqdm` not available.

See: https://stackoverflow.com/questions/15411967/how-can-i-check-if-code-is-executed-in-the-ipython-notebook

Reply:
You caught the review while I was still working on it. The current version has tqdm disabled unless a flag is passed to the session to enable it. We can add the conditional tqdm import as well if you want to min/max memory but the library boasts a very slim runtime.